net/net.go

```diff
@@ -720,14 +727,17 @@ func NewRecord(r core.Record, id thread.ID, lid peer.ID) (core.ThreadRecord, err
 	return &Record{Record: r, threadID: id, logID: lid}, nil
 }
 
+// Value returns the underlying value for the record.
```

Comment:
I generally leave off docstrings that are from implemented methods, since it's redundant... if you make a change then you have to make it in multiple places.